tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -260,6 +261,12 @@ func init() {
 	}
 	viper.SetDefault(constants.IptablesProbePort, strconv.Itoa(constants.DefaultIptablesProbePort))
 
+	rootCmd.Flags().String(constants.ProbeTimeout, strconv.FormatUint(constants.DefaultProbeTimeout, 10), "set timeout in nanosecond for failure detection")
```

Comment:
```suggestion
	rootCmd.Flags().Duration(constants.ProbeTimeout, constants.DefaultProbeTimeout, "failure detection timeout")
```

Reply:
TIL!

Reply:
FYI, this allows setting values like `1s`, `200ms`, etc. cf. https://golang.org/pkg/time/#ParseDuration